### DISCUSSION TITLE
probably a noob question

### DISCUSSION THREAD

**Discussion by tarun101** [Jason-Morcos/homebridge-magic-occupancy#44] [General]
I apologize if this is something obvious that I have not been able to figure out.
I think I get the gist of the six types of switches.

The question is: what is allowed to be put in these fields (for 6 types of switches) and in what format?
Am I supposed to put the string value as shown in “home” app? 
Also, home app has many different types of triggers different than “switches”: motion sensors, occupancy sensors, light, locks etc?
can any of them be placed in these 6 fields or are there any restrictions?

**Comment by Jason-Morcos** [comment]
I apologize for the delayed reply! You put the _name_ of each of the switches in the 6 fields. You can put a comma between names to create multiple of a given type.

All of the 6 types of switches are just "switches".  The plugin will create an occupancy sensor for each setup, one that responds to the combo of all the switches.
